migrations/1597956943277-lernstore_view_permission.js

```diff
@@ -0,0 +1,46 @@
+const { connect, close } = require('../src/utils/database');
+const Roles = require('../src/services/role/model');
```

Comment:
please instantiate a new model for the migration (as suggested in the migration template). Using a pre-compiled model twice (e.g. if another migration wants to use the role model as well) will lead to an error during `migration up`.

Reply:
thanks! I updated it, but am concerned a other migration files are like this